lxd/network/openvswitch/ovn.go

```diff
@@ -1946,6 +1946,10 @@ func (o *OVN) LoadBalancerApply(loadBalancerName OVNLoadBalancer, routers []OVNR
 			return fmt.Errorf("Missing VIP target(s)")
 		}
 
+		if len(args) > 0 {
+			args = append(args, "--")
```

Comment:
Could it be the case that the `--` gets added and a few lines down in `if r.Protocol == "udp"` the same gets appended again? 

I guess the resulting string would be `... -- -- lb-add ...`? Is this valid?

Reply:
Maybe I am not fully aware of the command syntax here.

Reply:
yeah i lose that bit

Reply:
fixed